Lesson-4-Task-2/src/test/java/Lesson4/Task2/CircleTest.java

```diff
@@ -0,0 +1,26 @@
+package Lesson4.Task2;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+public class CircleTest {
+    @Test
+    public void shouldAreaCircle(){
```

Comment:
should**Get**AreaCircle

w innych miejscach analogicznie

Reply:
Poprawione :)